juju/juju-gui
# Non-drag-and-drop unit placing workflow

This is the click-through unit placement workflow.

Question: can we use constraints for creating machines/containers? If so I need to add that, if not I can remove that for now.

QA:
Use il and mv flags.
- Drag a service to the canvas. Click Deploy and Confim.
- Drag a different service to the canvas. Do not deploy.
- Open the machine view.
- In the "Unplaced units" column there should be one unit. Click the icon on the right hand side.
- A select box should appear and you should now be able to follow through with adding to a machine/container, creating new machines/containers etc. All the paths should work.
